forte/pci/pci_sigma.h

```diff
@@ -58,7 +58,7 @@ class PCISigmaVector : public SigmaVector {
     void compute_sigma(psi::SharedVector sigma, psi::SharedVector b) override;
     void get_diagonal(psi::Vector& diag) override;
     void add_bad_roots(std::vector<std::vector<std::pair<size_t, double>>>& bad_states) override;
-    double compute_spin(const std::vector<double>& c) override { return 0.0; }
+    double compute_spin(const std::vector<double>& /*c*/) override { return 0.0; }
```

Comment:
I think this function should be fixed, maybe later.

Reply:
Correct. Left a TODO.